### ISSUE TITLE
Missing Information on pygeoapi INSTALL instructions

### ISSUE THREAD

**Issue by doublebyte1** [geopython/pygeoapi#1644]
**Description**
The [quick installation instructions](https://docs.pygeoapi.io/en/latest/installation.html#for-developers-and-the-truly-impatient) are missing some important information that may prevent people from installing pygeoapi.

**Steps to Reproduce**
Run [these](https://docs.pygeoapi.io/en/latest/installation.html#for-developers-and-the-truly-impatient) instructions.

**Expected behavior**
Based on this info, pygeoapi should be installed and run, without any errors. More details about the errors bellow.

**Additional context**
https://github.com/geopython/pygeoapi/pull/392/files#r1595349386
https://github.com/geopython/pygeoapi/pull/392/files#r1595351214

**Comment by tomkralidis** [comment]
@doublebyte1 I performed an installation as per https://docs.pygeoapi.io/en/latest/installation.html#for-developers-and-the-truly-impatient on a fresh ubuntu jammy image and it worked as expected.

Regarding your comments in https://github.com/geopython/pygeoapi/pull/392/files#r1595349386 and https://github.com/geopython/pygeoapi/pull/392/files#r1595351214:

>  pygeoapi won't build on python < 3.11 . We should indicate this on these instructions or else `python3 setup.py install` will fail.

Good point.  Added in #1760 for review.

> `example-openapi.yml` is not provided which will cause `pygeoapi serve` to fail. In order to get pygeoapi running, we should create at least an empty file : `echo > " " > example-openapi.yml`

`example-openapi.yml` is indeed not provided, but after setting the `PYGEOAPI_OPENAPI` environment variable, running `pygeoapi openapi generate $PYGEOAPI_CONFIG > $PYGEOAPI_OPENAPI` will create it accordingly (per the steps in the install doc).

Having said this, I've added a safeguard in #1758  to prevent `pygeoapi serve` from crashing on an undefined `PYGEOAPI_OPENAPI` environment variable, or if `PYGEOAPI_OPENAPI` refers to a file that is not found/available.
